src/Bookings.js

```diff
@@ -1,19 +1,77 @@
 import React, { Component } from "react";
 import Search from "./Search.js";
-// import SearchResults from './SearchResults.js';
-// import FakeBookings from '../data/fakeBookings.json';
+import SearchResults from "./SearchResults.js";
 
 export default class Bookings extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      FakeBookings: null,
+      isLoading: true
+    };
+  }
+
+  componentDidMount() {
+    fetch("https://cyf-react.glitch.me")
+      .then(res => {
+        if (res.status >= 200 && res.status < 300) {
+          return res;
+        } else {
+          throw new Error("HTTP error");
+        }
+      })
+      .then(res => res.json())
+      .then(data =>
+        this.setState({
+          FakeBookings: data,
+          isLoading: false,
+          SearchResults: data
```

Comment:
We generally say in React that it is better to calculate things when we need them rather than try to store everything in state. Here we only need to store the current value and the full list of results to be able to calculate the filtered list of results.

If we added a button to show all results again (after filtering), how would you implement that?

Hint: it might be easier to store just the current `searchVal` in state, then calculate the filtered results in the render method. Then if you want to show all the results again, you can just set the `searchVal` to be empty and then the filter will return all the results.